.claude/skills/radio-link-testing/scripts/tier1-downlink.sh

```
@@ -0,0 +1,292 @@
+  echo "[dl] ERROR: bridge port (21203) should have exactly 1 holder, got $bridge_holders"
+  echo "[dl]   Fix: pkill -f fprime_gds.executables.comm && bash /tmp/sweep/start_all_gds.sh"
+  exit 1
+fi
+echo "[dl]   bridge comm OK ($bridge_holders holder on /dev/cu.usbmodem21203)"
+
+# --- verify a received segment against ground truth ---
+# GDS writes bytes at their absolute source-file offset (SendPartial semantics):
+# seg_i file has OFFSET[i] zero bytes prepended, then LEN[i] bytes of real data.
+# We CRC the slice [OFFSET[i] : OFFSET[i]+LEN[i]] and require file >= that size.
+verify_seg(){ # idx -> 0 ok / 1 mismatch or missing
+  local i="$1" path="$DL_DIR/${DL_DEST[$i]}"
```

Shellcheck (0.11.0)

[warning] This assignment is used again in this 'local', but won't have taken effect. Use two 'local's.

(SC2318)